src/customFunctions/customFetch.ts

```diff
@@ -0,0 +1,15 @@
+import { bearerTokenAuthorization } from "./../configs/bearerTokenAuthorization";
+
+export const customFetch = async (
```

Comment:
takhle ta service-layer neškáluje do budoucna, pokuď někdo bude chtít víc customizovat http volání, doporučil bych se inspirovat `fetch` api a mít ho skoro totožné... takhle třeba volání customFetche, když člověk chce nastavit `mode`, ale je to GET, tak nemá `body`, tak to rozhraní je dost divné